content/docs/dvclive/api-reference/init.md

```diff
@@ -0,0 +1,63 @@
+# dvclive.init()
+
+Initializes a DVCLive logger.
+
+```py
+def init(
+    path: str = None,
+    resume: bool = False,
+    step: int = 0,
+    summary: bool = True,
+    html: bool = True,
+)
+```
+
+#### Usage:
+
+```py
+import dvclive
+
+dvclive.init()
+```
+
+## Description
+
+It's usage is optional and focused on configuring the behavior of subsequent
```

Comment:
BTW how is `init()` optional? I think an explanation on default values coming from a DVC environment is missing in this ref. And possibly which ones take precedence over the defaults indicated in the **Parameters** list?